src/main/java/use_case/Snowflake.java

```diff
@@ -0,0 +1,95 @@
+package main.java.use_case;
+
+/**
+ * Revised from Twitter's snowflake algorithm for generating unique IDs
+ * Github: https://github.com/twitter-archive/snowflake
```

Comment:
Maybe be more specific in the source
https://github.com/twitter-archive/snowflake/blob/b3f6a3c6ca8e1b6847baa6ff42bf72201e2c2231/src/main/scala/com/twitter/service/snowflake/IdWorker.scala

Reply:
OK changed